app/dao/fact_billing_dao.py

```diff
@@ -64,53 +64,52 @@ def fetch_sms_free_allowance_remainder_until_date(end_date):
 
 
 def fetch_sms_billing_for_all_services(start_date, end_date):
+    # ASSUMPTION: start and end date are in the same financial year
+    financial_year = get_financial_year_for_datetime(get_london_midnight_in_utc(start_date))
 
     # ASSUMPTION: AnnualBilling has been populated for year.
-    allowance_left_at_start_date_query = fetch_sms_free_allowance_remainder_until_date(start_date).subquery()
+    ft_billing_subquery = query_sms_usage_for_year_per_service(financial_year).subquery()
 
-    sms_billable_units = func.sum(FactBilling.billable_units * FactBilling.rate_multiplier)
+    sms_billable_units = func.sum(ft_billing_subquery.c.chargeable_units)
```

Comment:
yeah I agree with you but I'm concerend that on the org dashboard we refer to `sms_billable_units`. That comes from `fetch_usage_year_for_organisation`, which calls `fetch_sms_billing_for_organisation`, which calls this function.

i think best option might be to make it so that it stays consistent as chargeable until it gets to the dict conversion (since that feels like output formatting and manipulation rather than data gathering). thoughts?